RESTserver/pom.xml

```diff
@@ -13,45 +13,81 @@
     <packaging>war</packaging>
 
     <dependencies>
-
         <dependency>
             <groupId>com.softserve.if072</groupId>
             <artifactId>common-modules</artifactId>
-            <version>1.0-SNAPSHOT</version>
         </dependency>
 
-        <!-- https://mvnrepository.com/artifact/org.mybatis/mybatis -->
         <dependency>
-            <groupId>org.mybatis</groupId>
-            <artifactId>mybatis</artifactId>
-            <version>3.4.1</version>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
         </dependency>
 
         <dependency>
-            <groupId>org.mybatis</groupId>
-            <artifactId>mybatis-spring</artifactId>
-            <version>1.3.0</version>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
```

Comment:
Do not need to duplicate libs if they are mentioned in the parend pom.xml. Please remove here and in the MVCapplication/pom.xml